### PR TITLE
3.11 postrestorehook timeout config

### PR DESCRIPTION
**Pull request by sumanth-pasupuleti**

### CODE REVIEW

File: priam/src/main/java/com/netflix/priam/IConfiguration.java [hunk 1]
```diff
@@ -779,4 +779,16 @@
      * @return time out for post restore hook in days
      */
     public int getPostRestoreHookTimeOutInDays();
+
+    /**
+     * Heartbeat timeout (in ms) for post restore hook
+     * @return heartbeat timeout for post restore hook
+     */
+    public int getPostRestoreHookHeartBeatTimeoutInMs();
```
Comment (arunagrawal84) [inline priam/src/main/java/com/netflix/priam/IConfiguration.java]: please provide a default value in the interface, so we don't need to copy this method in all implementations of IConfiguration. Also, drop the `public` modifier as the interface is already `public`.

File: priam/src/test/java/com/netflix/priam/FakeConfiguration.java [hunk 2]
```diff
@@ -914,4 +914,12 @@ public String getPostRestoreHookDoneFileName() {
     public int getPostRestoreHookTimeOutInDays() {
         return 2;
     }
+
+    @Override
+    public int getPostRestoreHookHeartBeatTimeoutInMs() {
```
Comment (arunagrawal84) [inline priam/src/test/java/com/netflix/priam/FakeConfiguration.java]: No need to provide this if you provide default implementation in IConfiguration.

**Comment by sumanth-pasupuleti** [comment]
Thanks @arunagrawal84 . will be incorporating your comments and sending out a new PR (my branch ended up in a messy state)

**Comment by sumanth-pasupuleti** [comment]
Closing this PR and sending out another from a clean branch.

**Pull request closed without merging**